packages/instantsearch-ui-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "instantsearch-ui-components",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
can stay 0.1.0 in theory but that's fine as 0.2.0 as well (it's just internal anyway)

Reply:
I was not sure but it's probably good to have one release with shipjs when its not required by other packages just in case.

Reply:
Yes, I agree it's good to have a release with it, I just mean the version number could in theory be 0.1.0 as that's not released yet. Not worth the hassle changing though IMO